### PR TITLE
Nosql database

### PR DESCRIPTION
**Pull request by Latkos**
I did not check it working on the models, maybe someone could do that, but I can assure that Heroku can connect to the MongoDB Atlas database. It is not a secure solution for now though, as all

### CODE REVIEW

**Comment by pktiuk** [comment]
Może przy okazji dodawania bazy nosql powinieneś też dać opcję (i prostą instrukcję w readme) jak odpalić to na lokalnej instancji?

Abyśmy podczas developmentu mogli sobie u siebie pracować

**Comment by Latkos** [comment]
The only thing that is common between the development and production environments is the database itself. And you are right that it should not be that way, however I only see 3 options we can take from there:

1. We make a new development database on Atlas, which then means we only have to store an environment variable with the URI to the database on our computers (that solution is currently my favorite, though of course those databases are limited regarding storage etc.)
2. We leave this issue, periodically cleaning the database from clutter left over from development. It's not drastically important.
3. We let people have a local MongoDB instance. Then there are no limits etc. so that's a good solution - but it means everyone will be setting this up, which I do not want, purely because we're already behind...

**Comment by pktiuk** [comment]
I leave this decission up to you  
(IMO 3. is not so bad, because you can propably setup this base with 1 docker command)

File: apsi_diploma/apsi_diploma/settings.py [hunk 1]
```diff
@@ -9,29 +9,30 @@
...
-SECRET_KEY = "django-insecure-wv8nxxyjd)nvpc8^n#!ahui0nw+$kcbhi6!@bomr=g)t(2uoj6"
+SECRET_KEY = os.environ.get("DJANGO_APSI_DIPLOMA_SECRET_KEY")
 
 # SECURITY WARNING: don't run with debug turned on in production!
+
+if os.environ.get("IS_HEROKU_PRODUCTION"):
+    DEBUG = False
+else:
+    DEBUG = True
 DEBUG = False
```
Comment (pktiuk) [inline apsi_diploma/apsi_diploma/settings.py]: Nie zapomniałeś usunąć jednej linijki?

Comment (Latkos) [reply apsi_diploma/apsi_diploma/settings.py]: Yup

**Review by pktiuk — APPROVED** [review]

**Pull request merged**